utils/node-loader/src/batch_pool/batch/send_message.rs

```diff
@@ -2,11 +2,14 @@
 
 use crate::{
     batch_pool::Seed,
-    utils::{LoaderRng, NonEmptyVec, RingGet},
+    utils::{LoaderRng, NonEmptyVec},
 };
-use gear_core::ids::ProgramId;
+use gclient::Result as GClientResult;
+use gear_core::ids::{MessageId, ProgramId};
+use primitive_types::H256;
 
 pub type SendMessageArgsInner = (ProgramId, Vec<u8>, u64, u128);
+pub type SendMessageBatchOutput = (Vec<GClientResult<(MessageId, ProgramId)>>, H256);
```

Comment:
why don't you satisfied with just `Result` without any name mappings?

Reply:
`GClientResult` -> `Result`

Reply:
done